app/views/rentals/_form.html.erb

```diff
@@ -77,7 +76,7 @@
       <div class="form-group col-xs-12">
         <%= f.label :type, 'Cart Type', class: 'control-label col-xs-2' %>
         <div class="col-xs-6">
-          <%= f.select :item_type_id, options_from_collection_for_select(@item_types, "id", "name"), {}, class: "selectpicker form-control" %>
+          <%= f.select :item_type_id, options_from_collection_for_select(@item_types, "id", "name"), {}, class: "selectpicker form-control", onchange: "display_disc(this)" %>
```

Comment:
Rather than invoking this directly, register a listener.

I did not realize @owlee had declared the on change event from the ruby. Rather, you can do it within the JS with something like:

```Javascript
$( document ).ready(function() {
  $('input[type=whatever]').click/change/etc(function( {$(this).closest...
```

That way, all form element that change the cart type can use the same piece of JS, and you won't have any need for `, onchange: "display_disc(this)"`